### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by duhnnie**
Add Login feature to project

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "react-redux": "^5.1.1",
     "react-router": "^4.3.1",
     "react-router-dom": "^4.3.1",
+    "react-router-prop-types": "^1.0.4",
```
Comment (cinthia-neko) [inline package.json]: this library is repeated.

File: src/components/Login/Login.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    }
+  }
+
+  render () {
+    const { code } = getQueryParams(this.props.location.search);
+
+    return (
+        <div className='login-form'>
+          <Grid textAlign='center' style={{ height: '100%' }} verticalAlign='middle'>
+            <Grid.Column style={{ maxWidth: 450 }}>
```
Comment (veymarjoel) [inline src/components/Login/Login.js]: please use a different file for styles, inline styles shouldn't permitted

File: src/index.html [hunk 3]
```diff
@@ -4,6 +4,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
     <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/semantic-ui@2.4.0/dist/semantic.min.css" />
+    <style>
```
Comment (veymarjoel) [inline src/index.html]: please use a different file for styles

File: src/components/Login/Login.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      this.props.history.replace('/');
+    }
+  }
+
+  render () {
+    const { code } = getQueryParams(this.props.location.search);
+
+    return (
+        <div className='login-form'>
+          <Grid textAlign='center' style={{ height: '100%' }} verticalAlign='middle'>
```
[the remark this answers is not in the document]
Comment (duhnnie) [reply src/components/Login/Login.js]: Done, it was moved to a .css file.

**Comment by duhnnie** [comment]
The PR was updated with changes

**Pull request merged**